hexa/workspaces/graphql/schema.graphql

```diff
@@ -9,8 +9,31 @@ type Workspace {
   updatedAt: DateTime
   createdBy: User!
   connections: [Connection!]!
+  invitations(status: WorkspaceInvitationStatus,page: Int, perPage: Int): WorkspaceInvitationPage!
 }
 
+type WorkspaceInvitationPage {
+  pageNumber: Int!
+  totalPages: Int!
+  totalItems: Int!
+  items: [WorkspaceInvitation!]!
+}
+
+type WorkspaceInvitation {
+  id: UUID!
+  email: String!
+  workspace: Workspace!
+  role: WorkspaceMembershipRole!
+  status: WorkspaceInvitationStatus!
+  invited_by: User!
```

Comment:
Since the user of a invitation can be deleted (user is deleted from the database), this field is nullable